presto-common/src/test/java/com/facebook/presto/common/type/TestTimeZoneKey.java

```diff
@@ -47,88 +47,79 @@ public void testUTC()
 
         // verify UTC equivalent zones map to UTC
         assertSame(TimeZoneKey.getTimeZoneKey("Z"), UTC_KEY);
-        assertSame(TimeZoneKey.getTimeZoneKey("Zulu"), UTC_KEY);
-        assertSame(TimeZoneKey.getTimeZoneKey("zulu"), UTC_KEY);
         assertSame(TimeZoneKey.getTimeZoneKey("ZULU"), UTC_KEY);
         assertSame(TimeZoneKey.getTimeZoneKey("UT"), UTC_KEY);
         assertSame(TimeZoneKey.getTimeZoneKey("UCT"), UTC_KEY);
         assertSame(TimeZoneKey.getTimeZoneKey("Universal"), UTC_KEY);
-        assertSame(TimeZoneKey.getTimeZoneKey("GMT"), UTC_KEY);
-        assertSame(TimeZoneKey.getTimeZoneKey("GMT0"), UTC_KEY);
-        assertSame(TimeZoneKey.getTimeZoneKey("GMT+0"), UTC_KEY);
-        assertSame(TimeZoneKey.getTimeZoneKey("GMT-0"), UTC_KEY);
-        assertSame(TimeZoneKey.getTimeZoneKey("GMT+00:00"), UTC_KEY);
-        assertSame(TimeZoneKey.getTimeZoneKey("GMT-00:00"), UTC_KEY);
+        assertTimeZoneNotSupported("GMT+00:00");
+        assertTimeZoneNotSupported("GMT-00:00");
         assertSame(TimeZoneKey.getTimeZoneKey("+00:00"), UTC_KEY);
         assertSame(TimeZoneKey.getTimeZoneKey("-00:00"), UTC_KEY);
-        assertSame(TimeZoneKey.getTimeZoneKey("etc/utc"), UTC_KEY);
-        assertSame(TimeZoneKey.getTimeZoneKey("etc/gmt"), UTC_KEY);
+        assertTimeZoneNotSupported("etc/utc");
+        assertTimeZoneNotSupported("etc/gmt");
```

Comment:
Both of these entries are found here and should be supported: https://en.wikipedia.org/wiki/List_of_tz_database_time_zones